source/lovelace/views.markdown

```diff
@@ -9,88 +9,115 @@ sharing: true
 footer: true
 ---
 
-These are exactly as before, tab views with icons or text that help you manage large dashboards with many entities. The views have now deep-links like `/lovelace/0`. You can also assign your own [custom IDs](/lovelace/views/#custom-id).
-
-- Using custom id in view, for [nicer navigation paths](/lovelace/views/#custom-id) in URLs.
-- Using [icons](/lovelace/views/#icons) instead of text.
-- Using a card to [fill a complete view](/lovelace/views/#panel-view), just like panels.
-- Using [themes](/lovelace/views/#themes) in views.
-
-| Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
-| title | string | Optional | Text title of the view
-| id | string | number | The id to use in URL path of this view
-| icon | string | Optional | The material design icon for the view, uses this instead of title
-| panel | boolean | false | Marks view as a panel reusing the first card in the list
+{% configuration %}
+views:
+  required: true
+  description: A list of view configurations.
+  type: list
+  keys:
+    title:
+      required: true
+      description: A title for this view.
+      type: string
+    cards:
+      required: true
+      description: List of cards to render in this view.
+      type: list
+    id:
```

Comment:
I believe this needs to be "id" to validate with netlify.

Reply:
:church: